### PR TITLE
Add an Experiment page

### PR DESCRIPTION
**Pull request by smpadhy**
Experiment page allows a user to add an experiment, enters information and choose a template file from existing project directory or in different directory. It saves the information in JSON format.
It implements the design shown in Slide 4 in the design document, Issue #5.

### CODE REVIEW

File: src/addExperiment.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            <li><a href="#">Executions</a></li>
+          </ul>
+          </div>
+      </div>
+        <div class="col-xs-9">
+          <div class="">
+            <p><h1>Add an Experiment</h1></p>
+            <div>
+              <form>
+                <div class="form-group row">
```
Comment (satra) [inline src/addExperiment.html:63]: the pieces of the form should come from a json dictionary (which in the future would come from nidm-experiment)

File: src/addProject.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>BrainVerse</title>
+
+    <!-- Bootstrap -->
+    <link href="../node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/brainverse.css" rel="stylesheet">
+
+  </head>
+  <body>
```
Comment (satra) [inline src/addProject.html:15]: same for projects as well. the basic idea should be there is a jsonld data structure, that feeds an html template to generate the view.

File: src/addProjectTerms.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "Acquisition" : [
+    "MRI Acquisition",
+    "Demographics"
+  ],
+  "Project Information" : [
+    "Publications",
+    "Funding Sources",
+    "Collaborators"
+  ]
+}
```
Comment (satra) [inline src/addProjectTerms.json:34]: so indeed something like this should drive the html render.

**Review by satra — APPROVED** [review]
i left comments for future iterations.

File: src/addExperiment.html (hunk 1 above)
Comment (smpadhy) [reply src/addExperiment.html:63]: Yes. However, we need to decide as which entity/activity/agent class/subclass in nidm experiment we would provide within the form to the user.

File: src/addProject.html (hunk 2 above)
Comment (smpadhy) [reply src/addProject.html:15]: yes. Issue #7 aims  to create a parser for NIDM experiment provided in one of the formats OWL/turtle/jsonld. Obtains the lexicon from the file and will be feeds to the html template.

**Pull request merged**